paddle/operators/recv_op.cc

```diff
@@ -57,7 +59,10 @@ class RecvOp : public framework::OperatorBase {
     }
   }
 
-  virtual ~RecvOp() {
+  void Stop() override {
```

Comment:
Maybe we also need the destructor function, and call `Stop()` in it.

Reply:
No. `recv_op`'s `Run` may run in another thread (in unit test), `Stop` must be called out side dtor

Reply:
So we also need to add the interface to `pybind` , maybe we coud do that in another PR.

Reply:
Yep. We can add it later when running in different clusters.